nevergrad/optimization/optimizerlib.py

```diff
@@ -1668,3 +1717,36 @@ def __init__(self, parametrization: IntOrParameter, budget: Optional[int] = None
                     self.optims = [CMA(self.parametrization, budget, num_workers)]
             else:
                 self.optims = [NGO(self.parametrization, budget, num_workers)]
+
+
+@registry.register
+class MetaModel(base.Optimizer):
+    """Adding a metamodel into CMA."""
+
+    def __init__(self, parametrization: IntOrParameter, budget: Optional[int] = None, num_workers: int = 1) -> None:
+        super().__init__(parametrization, budget=budget, num_workers=num_workers)
+        assert budget is not None
+        self.optims = [
```

Comment:
Please don't use a list for only one optimizer.
Like in Shiwa, there is no standard behavior if there are several optimizers, so this is not useful, better provide one optimizer, which itself may be a chaining or portfolio or whatever.

Reply:
Modified.